### DISCUSSION TITLE
Autocomplete: disable auto close?

### DISCUSSION THREAD

**Discussion by sebj54** [formkit/formkit#1523] [Ideas]
Hi there! 

I am implementing the autocomplete in my website currently and I would like to disable the "auto close" feature. It is pretty annoying combined with the "multiple" mode because you have to open the dropdown for each option you'd like to select.

Is there a way to disable this behaviour? I did not found it in the docs. And if not, is it possible to add it? 🙏

**Comment by justin-schroeder** [comment]
You can use the `close-on-select` prop: https://formkit.link/cdd97e73d870136d985d833181881889

**Reply by sebj54** [reply]
Seems I missed it! Thanks for the tip :)
